src/custom/pages/Faq/index.tsx

```diff
@@ -415,6 +462,39 @@ export default function Faq() {
             allow you to wrap and unwrap ETH into WETH without needing to leave the dapp’s UI
           </p>
 
+          <h3 id="why-is-selling-eth-more-troublesome">
+            Why is selling ETH more troublesome?
+          </h3>
+
+          <p>
+            CowSwap only operates with ERC20 tokens. ETH is the native Ethereum currency, which is not an ERC20 token.
```

Comment:
May be worth including the technical limitation that you cannot use off-chain signatures for specifying an ETH allowance or transfer.

It's fine as is though.